alarm/kodi-c1/PKGBUILD

```diff
@@ -7,6 +7,7 @@
 # Contributor: BlackIkeEagle < ike DOT devolder AT gmail DOT com >
 # Contributor: Bartłomiej Piotrowski <bpiotrowski@archlinux.org>
 # Contributor: Maxime Gauduin <alucryd@gmail.com>
+# Contributor: Jan Holthuis <holthuis.jan@googlemail.com>
```

Comment:
I thought you do this everytime you change a package. If you don't want that, I'll leave it out. I'm not that vain ;-)